src/main/js/helpers/constants.js

```diff
@@ -213,7 +219,7 @@ export default {
     DEFAULT_REGION_LINE_STROKE_WIDTH: 5,
     DEFAULT_BAR_SELECTION_PADDING: 5,
     DEFAULT_ACTIVITY_BAR_HASH_COLOR: "#d3d4d5",
-    DEFAULT_TASK_BAR_HASH_COLOR: "#007cc3",
+    DEFAULT_TASK_BAR_HASH_COLOR: "#176ba0",
```

Comment:
Can we add before and after screenshot for this color change in Bar graph?

Reply:
and also we need to confirm with the UX team for this change.

Reply:
Here #176ba0 is hex code for blue color. I guess i already included how blue color look like, before and after changes.

Reply:
I meant just `DEFAULT_TASK_BAR_HASH_COLOR`? what is Bar hash color? where do we use color?

Reply:
Updated images as needed. Please see the description for screenshots.